builtin/tests/commit_post_test.rs

```diff
@@ -0,0 +1,649 @@
+    seal_proof: RegisteredSealProof,
+    _owner: Address,
+    worker: Address,
+    miner_id: Address,
+    _miner_robust: Address,
+}
+
+struct BlockstoreWrapper<'a>(&'a dyn Blockstore);
+
+impl<'a> Blockstore for BlockstoreWrapper<'a> {
+    fn get(&self, k: &Cid) -> anyhow::Result<Option<Vec<u8>>> {
+        self.0.get(k)
+    }
+
+    fn put_keyed(&self, k: &Cid, block: &[u8]) -> anyhow::Result<()> {
+        self.0.put_keyed(k, block)
+    }
+}
+
+pub fn make_sealed_cid(input: &[u8]) -> Cid {
+    make_cid_poseidon(input, FIL_COMMITMENT_SEALED)
+}
+
+pub fn make_cid_poseidon(input: &[u8], prefix: u64) -> Cid {
+    make_cid(input, prefix, MhCode::PoseidonFake)
+}
+
+pub fn sector_deadline(w: &mut ExecutionWrangler, m: &Address, s: SectorNumber) -> (u64, u64) {
+    let m = w.resolve_address(m).unwrap().unwrap();
+    let st: MinerState = w.find_actor_state(m).unwrap().unwrap();
+    st.find_sector(&Policy::default(), &BlockstoreWrapper(w.bench.borrow().store()), s).unwrap()
+}
+
+pub fn advance_to_proving_deadline(
+    w: &mut ExecutionWrangler,
+    maddr: &Address,
+    s: SectorNumber,
+) -> (DeadlineInfo, u64) {
+    let (d, p) = sector_deadline(w, maddr, s);
+    let dline_info = advance_by_deadline_to_index(w, maddr, d);
+    w.set_epoch(dline_info.open);
+    (dline_info, p)
+}
+
+pub fn advance_by_deadline_to_index(
+    w: &mut ExecutionWrangler,
+    maddr: &Address,
+    i: u64,
+) -> DeadlineInfo {
+    advance_by_deadline(w, maddr, |dline_info| dline_info.index != i)
+}
+
+#[allow(clippy::too_many_arguments)]
+pub fn precommit_sectors_v2(
+    w: &mut ExecutionWrangler,
+    count: u64,
+    batch_size: i64,
+    worker: &Address,
+    maddr: &Address,
+    seal_proof: RegisteredSealProof,
+    sector_number_base: SectorNumber,
+    expect_cron_enroll: bool,
+    exp: Option<ChainEpoch>,
+    v2: bool,
+) -> Vec<SectorPreCommitOnChainInfo> {
+    let mid = w.resolve_address(maddr).unwrap().unwrap();
+    // let invocs_common = || -> Vec<ExpectInvocation> {
+    //     vec![
+    //         ExpectInvocation {
+    //             to: REWARD_ACTOR_ADDR,
+    //             method: RewardMethod::ThisEpochReward as u64,
+    //             ..Default::default()
+    //         },
+    //         ExpectInvocation {
+    //             to: STORAGE_POWER_ACTOR_ADDR,
+    //             method: PowerMethod::CurrentTotalPower as u64,
+    //             ..Default::default()
+    //         },
+    //     ]
+    // };
+    // let invoc_first = || -> ExpectInvocation {
+    //     ExpectInvocation {
+    //         to: STORAGE_POWER_ACTOR_ADDR,
+    //         method: PowerMethod::EnrollCronEvent as u64,
+    //         ..Default::default()
+    //     }
+    // };
+    // let invoc_net_fee = |fee: TokenAmount| -> ExpectInvocation {
+    //     ExpectInvocation {
+    //         to: BURNT_FUNDS_ACTOR_ADDR,
+    //         method: METHOD_SEND,
+    //         value: Some(fee),
+    //         ..Default::default()
+    //     }
+    // };
```

Comment:
You can just delete all these expectations parts for now. We'll get them back when we import the tests directly